utils/format.go

```diff
@@ -859,87 +843,78 @@ func FormatTimestampTs(ts time.Time) template.HTML {
-		if len(tagName) > 1 {
-			_, err := strconv.ParseInt(tagName[len(tagName)-1], 10, 64)
-			if err == nil {
-				name := ""
-				for _, s := range tagName[:len(tagName)-1] {
-					if s == "-" {
-						continue
-					}
-					name += s + " "
-				}
-				return fmt.Sprintf(`<a href='/pools' style="all: unset; cursor: pointer;" data-toggle="tooltip" title="This validator is part of a staking-pool"><span style="font-size: 18px;" class="bg-light text-dark badge-pill pr-2 pl-0 mr-1"><span class="bg-dark text-light rounded-left mr-1 px-1">%s</span> %s</span></a>`, tagType, name)
-			}
-		}
-		return fmt.Sprintf(`<a href='/pools' style="all: unset; cursor: pointer;" data-toggle="tooltip" title="This validator is part of a staking-pool"><span style="font-size: 18px;" class="bg-light text-dark badge-pill pr-2 pl-0 mr-1"><span class="bg-dark text-light rounded-left mr-1 px-1">%s</span> %s</span></a>`, tagType, tag[len(tag)-1])
-	}
-	return ""
-}
-
-func formatSpecialTag(tag string) string {
-	special_tag := strings.Split(tag, ":")
-	if len(special_tag) > 1 {
-		if special_tag[0] == "pool" {
-			return formatPool(special_tag)
-		}
+		return `<span class="text-danger">Slashed</span>`
 	}
-	return fmt.Sprintf(`<span style="font-size: 18px;" class="badge bg-dark text-light mr-1">%s</span>`, tag)
+	return "Unknown"
 }
 
 // FormatValidatorTag will return html formated text of a validator-tag.
 // Depending on the tag it will describe the tag in a tooltip and link to more information regarding the tag.
-func FormatValidatorTag(tag string) template.HTML {
+func formatValidatorTag(tag string) string {
 	var result string
-	switch tag {
-	case "rocketpool":
-		result = `<span style="background-color: rgba(240, 149, 45, .2); font-size: 18px;" class="badge-pill mr-1 font-weight-normal" data-toggle="tooltip" title="Rocket Pool Validator"><a style="color: var(--yellow);" href="/pools/rocketpool">Rocket Pool</a></span>`
-	case "ssv":
-		result = `<span style="background-color: rgba(238, 113, 18, .2); font-size: 18px;" class="badge-pill mr-1 font-weight-normal" data-toggle="tooltip" title="Secret Shared Validator"><a style="color: var(--orange);" href="https://github.com/bloxapp/ssv/">SSV</a></span>`
+	switch {
+	case tag == "rocketpool", tag == "pool:rocketpool":
+		result = `<a href="/pools/rocketpool" title="Rocket Pool Validator" data-toggle="tooltip" class="mr-2 badge badge-pill style-bg-neutral-1 style-status-tag-text text-dark">Rocket Pool</a>`
+	case strings.HasPrefix(tag, "pool:"):
+		result = fmt.Sprintf(`<a href="/pools" title="This validator is part of a staking-pool." data-toggle="tooltip" class="mr-2 badge badge-pill style-bg-neutral-1 style-status-tag-text text-dark">Pool: %s</a>`, tag[5:])
+	case strings.HasPrefix(tag, "name:"):
+		result = fmt.Sprintf(`<span title="This name has been set by the owner of this validator." data-toggle="tooltip" class="mr-2 badge badge-pill style-bg-neutral-1 style-status-tag-text text-dark">%s</span>`, tag[5:])
```

Comment:
Using `tag[5:]` to trim "pool:" and "name:" is really hard to read and you have to count characters, how about a `strings.TrimPrefix(tag, "pool/name:")`